.env.development

```
@@ -1,7 +1,11 @@
 # MSW (Mock Service Worker) 설정
 # true: MSW 활성화 (목 데이터 사용)
 # false: MSW 비활성화 (실제 API 사용)
-VITE_MSW_ENABLED=true
+
 
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)